platform/security/handler/security-handler-oidc/src/main/java/org/codice/ddf/security/handler/oidc/OidcCallbackEndpoint.java

```diff
@@ -54,12 +58,36 @@ public Response logout(
 
     sessionStore.destroySession(j2EContext);
 
+    String localLogout = SystemBaseUrl.EXTERNAL.constructUrl("/logout/local");
+    WebClient webClient = getWebClient(localLogout);
+    Response logoutResponse = webClient.get();
+    if (logoutResponse.getStatus() == HttpStatus.SC_INTERNAL_SERVER_ERROR) {
+      return logoutResponse;
+    }
+
     try {
-      return Response.temporaryRedirect(new URI(SystemBaseUrl.EXTERNAL.constructUrl("/logout")))
-          .build();
+      String redirectUrl = SystemBaseUrl.EXTERNAL.constructUrl(redirectUri, false);
```

Comment:
❓ Why exclude the root context?

Reply:
The url is `/logout/logout-response` it doesn't have `/services`

Reply:
Got it. Was confusing root/external context